indico/core/signals/__init__.py

```diff
@@ -14,5 +14,7 @@
 # You should have received a copy of the GNU General Public License
 # along with Indico; if not, see <http://www.gnu.org/licenses/>.
 
-from indico.core.signals import acl, agreements, attachments, category, event, event_management, plugin, users
+from indico.core.signals import (acl, agreements, attachments, category, event, event_management, menu, plugin, users)
 from indico.core.signals.core import *
+
+__all__ = ('acl', 'agreements', 'attachments', 'category', 'event', 'event_management', 'menu', 'plugin', 'users', 'rb')
```

Comment:
Yeah, I get it. But that's only for core signals. If you add a new signal file you'll have to come here.
Why can't we just add a `core` namespace, BTW? It seems weird to me that some signals are located one level above.